src/layouts/index.jsx

```diff
@@ -144,6 +145,35 @@ const Layout = ({ children, location }) => {
         </div>
         <Sidebar location={location} />
       </div>
+      <CookieConsent
+        className="mama"
+        disableStyles="true"
+        location="bottom"
+        buttonText="I agree"
+        declineButtonText="Decline"
+        cookieName="gatsby-gdpr-google-analytics"
+      >
+        <p>
+          This website uses cookies to ensure you get the best browsing

```

Comment:
This and next row: The “{" "}” parts don’t seem to be part of the output, but are they needed?